adal/src/main/java/com/microsoft/aad/adal/ADALError.java

```diff
@@ -36,577 +42,567 @@
     /**
      * Authority validation returned an error.
      */
-    DEVELOPER_AUTHORITY_CAN_NOT_BE_VALIDED(com.microsoft.identity.common.adal.error.ADALError.DEVELOPER_AUTHORITY_CAN_NOT_BE_VALIDED),
+    DEVELOPER_AUTHORITY_CAN_NOT_BE_VALIDED("Authority validation returned an error"),
```

Comment:
@shoatman Was the developer who had originally migrated these errors into `common` -- I suspect he'll also want to review these changes once he returns.